thentos-core/src/ThentosDocs.hs

```diff
@@ -0,0 +1,13 @@
+module ThentosDocs (makeMain) where
+
+import Data.Thyme.Time ()
```

Comment:
ah.  good question.  hm...  probably used to be used somewhere, but isn't any more?  i'll see if i can remove it, and PR elsewhere.